src/app/EventManagement.cpp

```diff
@@ -610,20 +610,20 @@ CHIP_ERROR EventManagement::CopyEvent(const TLVReader & aReader, TLVWriter & aWr
 
 static bool IsInterestedEventPaths(EventLoadOutContext * eventLoadOutContext, const EventEnvelopeContext & event)
 {
-    ClusterInfo * interestedEventPaths = eventLoadOutContext->mpInterestedEventPaths;
+    ClusterInfo * interestedEventPath = eventLoadOutContext->mpInterestedEventPaths;
```

Comment:
```suggestion
for (auto * interestedPath = eventLoadOutContext->mpInterestedEventPaths; interestedPath != nullptr; interestedPath = interestedPath->Next)
```

And move to the line after `if`

Reply:
Done, thanks